### PR TITLE
BookA-Pset_InSituTestCPT

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_InSituTestCPT

## Description: 

Properties for the results of Cone Penetration Tests/Cone Penetrometer Tests (CPTs).

| Property | Datatype | Description |
|----------|----------|-------------|
|CPTConeResistance|IfcPressureMeasure|Cone reistance measured during the test.|
|CPTLocalSideFriction|IfcPressureMeasure|Sleeve friction measured during the test.|
|CPTPorePressure|IfcPressureMeasure|Pore pressure measured during the test.|
|YoungModulus|According to existing definition|According to existing definition|

### CODE REVIEW

**Comment by patmclarin** [comment]
please update the Pset description to: Properties for the results of Cone Penetration Tests/Cone Penetrometer Tests (CPTs)

File: IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Emodulus_1Bctvx1L570B_eZVJbTFB8" Name="Emodulus" UniqueId="4b9b7e7b-0551-4700-bfa8-8df4e574f2c8" PrimaryDataType="IfcModulusOfElasticityMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml]: Use YoungModulus.

Comment (larswik) [reply IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml]: > YoungModulus

As a term (property name) you mean? From my vague recollection, Young Modulus refers to tensile elasticity. @j-weil

**Comment by larswik** [comment]
> please update the Pset description to: Properties for the results of Cone Penetration Tests/Cone Penetrometer Tests (CPTs)

Done.

File: IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/e/Emodulus_1Bctvx1L570B_eZVJbTFB8/DocProperty.xml]: Fixed in [feab7eb](https://github.com/bSI-InfraRoom/IFC-Specification/pull/608/commits/feab7ebdc7ff4eb8297c47cf0d8b44f998b4018b)

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**